docs/question/index.md

```diff
@@ -141,104 +176,62 @@ The `question.html` is a template used to render the question to the student. A
 
 The `question.html` is regular HTML, with four special features:
 
-1. Any text in double-curly-braces (like `{{params.m}}`) is substituted with variable values. If you use triple-braces (like `{{{params.html}}}`) then raw HTML is substituted (don't use this unless you know you need it). This is using [Mustache](https://mustache.github.io/mustache.5.html) templating.
+1. Any text in double-curly-braces (like `{{params.m}}`) is substituted with variable values using [Mustache](https://mustache.github.io/mustache.5.html). These values typically are defined by a [question's `server.py`](#custom-generation-and-grading-serverpy).
```

Comment:
I think this discussion should also be added to the html.md document, the current version seems quite eager to use advance Mustache features before describing the basics.

Reply:
I somewhat agree, but some of this information seems important "at a glance". I'm not sure the best way to handle this -- additionally, this list fails to mention the different panels of a question.

Reply:
To be clear: I don't mean move it there, I mean copy it (or something similar) there. I believe the html.md should start with simpler examples using `{{params.xyz}}` instead of moving directly to conditional rendering.

Reply:
Gotcha, I will do that.

Reply:
Changed in d26f5c0ebf3dae1327cabfcedc8189dc52f12366. Let me know what you think!